hieradata/site/ls/role/ipareplica.yaml

```diff
@@ -0,0 +1,3 @@
+---
+easy_ipa::configure_replica_ca: true
+easy_ipa::ipa_master_fqdn: "ipa1.cp.lsst.org"
```

Comment:
Should this be ipa1.ls.lsst.org?

Reply:
As above, this is the IPA master that the replica attaches to.

Reply:
Doh, yes, of course.  However, I believe `ipa1.tuc.lsst.org` is currently the master?

Reply:
Once a domain replica has been added, it has the same rights and capabilities as any other master. `man ipa-replica-install` says the following:

```
       ipa-replica-install - Create an IPA replica

SYNOPSIS
   DOMAIN LEVEL 0
       ipa-replica-install [OPTION]... [replica_file]

   DOMAIN LEVEL 1
       ipa-replica-install [OPTION]...

DESCRIPTION
       Configures  a new IPA server that is a replica of the server. Once it has been created it is an
       exact copy of the original IPA server and is an equal master. Changes made to  any  master  are
       automatically replicated to other masters.
```

Right now the `139.229.135.0/24` subnet can't directly reach ipa1.tuc.lsst.cloud so I've attached ipa1.ls.lsst.org to ipa1.cp.lsst.org. I've promoted ipa1.cp.lsst.org to a CA master as well and have replicated the CA to ipa1.ls.lsst.org from there.

![Screenshot from 2020-02-20 15-37-05](https://user-images.githubusercontent.com/172194/74967043-fd99d980-53f6-11ea-9c48-15a655eb8c5c.png)

Reply:
I was not aware that the CA has been replicated. I've opened https://jira.lsstcorp.org/browse/IT-1796 to make sure this becomes part of the puppet configuration.

Reply:
This is being handled by Puppet on new nodes:

https://github.com/lsst-it/lsst-itconf/pull/40/files#diff-032e052f37da931d7d6e365ac337d9ddR2